integration/spark2/src/main/scala/org/apache/carbondata/indexserver/DataMapJobs.scala

```diff
@@ -118,7 +142,15 @@ class EmbeddedDataMapJob extends AbstractDataMapJob {
     }
     dataMapFormat.setTaskGroupId(taskGroupId)
     dataMapFormat.setTaskGroupDesc(taskGroupDesc)
-    IndexServer.getSplits(dataMapFormat).toList.asJava
+    val splits = try {
+      IndexServer.getSplits(dataMapFormat)
+        .getExtendedBlockets(dataMapFormat.getCarbonTable.getTablePath, dataMapFormat.getQueryId)
+    } finally {
+      if (null != tmpFolder && !tmpFolder.delete()) {
+        LOGGER.info("Problem while deleting the temp directory" + tmpFolder.getAbsolutePath)
```

Comment:
1. change it to `Logger.error`
2. Add space after the constant message

Reply:
fixed